ui/src/components/erc/ERC.js

```diff
@@ -213,6 +224,28 @@ class ERC extends React.Component {
 
 
     handleTabChange = (e, newValue) => {
+
+    
+        let hash = "#"
+        switch(newValue){
```

Comment:
This will have to be changed when a new tab is added, right?
Can you reuse the values "Check", "Manipulate" etc. somehow? Or can you try to define them only once and reuse them both here and where the tab titles are defined?

Reply:
Right, this has to be changed, when a tab is added. I will try to define them only once. I think it is not possible to reuse them.

Reply:
Good!